src/ImageSharp/Memory/Allocators/Internals/SharedArrayPoolBuffer{T}.cs

```diff
@@ -74,7 +79,6 @@ protected override void Release()
             // meaning likely a different bucket than it was rented from,
             // but this is PROBABLY better than not returning the arrays at all.
             ArrayPool<byte>.Shared.Return(this.array);
-            this.array = null;
```

Comment:
It's better to make `this.array` nullable so it's detached after a call to `Release`.